openmc/universe.py

```diff
@@ -360,12 +360,15 @@ def plot(self, origin=(0., 0., 0.), width=(1., 1.), pixels=(200, 200),
 
             # Create a figure sized such that the size of the axes within
             # exactly matches the number of pixels specified
-            px = 1/plt.rcParams['figure.dpi']
-            fig, ax = plt.subplots()
-            params = fig.subplotpars
-            width = pixels[0]*px/(params.right - params.left)
-            height = pixels[0]*px/(params.top - params.bottom)
-            fig.set_size_inches(width, height)
+            if kwargs.get("ax") is None:
```

Comment:
Rather than pulling this out of `kwargs`, let's just add an `axes` argument for this method, in which case this would become:
```python
if axes is None:
    px = 1/plt.rcParams['figure.dpi']
    fig, axes = plt.subplots()
    ...

return axes.imshow(...)
```